### DISCUSSION TITLE
Focus trap on Modal with custom layout

### DISCUSSION THREAD

**Discussion by nicolaidanekov** [Blazored/Modal#398] [Q&A]
Hey there!

I am trying to implement a Blazored Modal with a Bootstrap look and feel.
I am using the ModalService with the options "var options = new ModalOptions {UseCustomLayout = true};" as described here: https://github.com/Blazored/Modal/pull/189

This allows me to create something that looks like Bootstrap, but it means I lose everything that is awesome about the Blazored Modal - including Focus trap.

Is there a way for me to use focus trap on a modal with a custom layout? Or is there possibly an easier way for me to achieve Bootstrap layout while using the Blazored Modal?

@chrissainty @dfkeenan 

Best regards,
Nicolai

**Comment by nicolaidanekov** [comment]
I was able to borrow your FocusTrap.razor and wrap my modal in it. That worked perfectly. 👍

**Reply by chrissainty** [reply]
Glad you got it working! 🙂
